### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Gredin67**
## Problem
- *Description of why you made this PR*

## Solution
- *And how do you fix that problem*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/APP_ynh%20PR-NUM-%20(USERNAME)/)

### CODE REVIEW

**Comment by Gredin67** [comment]
!testme

**Comment by oufmilo** [comment]
!testme

**Comment by oufmilo** [comment]
It's a manifest problem but i don't understand ..

**Comment by Gredin67** [comment]
I  really don't get what is happening here with process check https://ci-apps-dev.yunohost.org/ci/job/2177
``&bot_is_synapse_admin=package_checker``
@alexAubin

**Comment by Gredin67** [comment]
for the upgrade failing, we obviously need to remove more files before upgrading, or to git pull in some folder https://github.com/tulir/mautrix-whatsapp/issues/317
The fresh install failing because of missing config file. I tried to add a sleep time to wait for the config file to be generated. Otherwise we have to understand why ynh_add_config does not generate the file properly https://paste.yunohost.org/raw/nalaqewiwo

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
!testme

**Comment by Gredin67** [comment]
@tituspijean could you have a look at this PR, I tried maaany things but the ``go build`` does not work with the upgraded sources.
The standard procedure is to do ``git pull`` rather than downloading new sources. And the maintainer thinks it could be related. https://github.com/tulir/mautrix-whatsapp/issues/317

I am thinking of not using ``ynh_setup_sources`` anymore and rather doing a ``git checkout v0.1.7`` to see what it gives..

**Comment by Gredin67** [comment]
!testme

**Review by oufmilo — CHANGES_REQUESTED** [review]
;;; Upgrade options
	; commit=636be2f9f900e578d9aa26ee887dcec2ef8d4e71
		name=0.1.7

File: check_process [hunk 1]
```diff
@@ -37,21 +18,10 @@
...
-		# incorrect_path=1
 		port_already_use=1
 		change_url=0
-;;; Levels
-	# If the level 5 (Package linter) is forced to 1. Please add justifications here.
-	Level 5=auto
-;;; Options
-#Email=
-Notification=none
-;;; Upgrade options
```
Comment (oufmilo) [inline check_process:53]: ;;; Upgrade options
	; commit=636be2f9f900e578d9aa26ee887dcec2ef8d4e71
		name=0.1.7

**Review by oufmilo — CHANGES_REQUESTED** [review]

File: check_process [hunk 2]
```diff
@@ -37,21 +18,10 @@
...
-	; commit=CommitHash
```
Comment (oufmilo) [inline check_process:54]: ; commit=636be2f9f900e578d9aa26ee887dcec2ef8d4e71

File: check_process (hunk 1 above)
Comment (oufmilo) [inline check_process:53]: ;;; Upgrade options

File: check_process [hunk 3]
```diff
@@ -37,21 +18,10 @@
...
-		#name=Name and date of the commit.
```
Comment (oufmilo) [inline check_process:55]: name=0.1.7

**Review by oufmilo — CHANGES_REQUESTED** [review]
check_process update

**Comment by oufmilo** [comment]
!testme

**Pull request merged**